components/support/nimbus-fml/ExperimentFeatureManifest.schema.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
reminder to myself to think about this a little, I **THINK** this is okay to keep here, but we have to be conscious of what happens if the schema changes in moz-central, I don't think it should fail the tests, especially considering others who aren't working on this will be hit by a failure.

That said I might setup a CI job that validates the current schema here vs the schema on moz-central and if they are different, it opens an issue on github for us to go get it fixed

This defs won't be a part of the work here, but I'll file a follow-up if we think that's a good approach

Reply:
Is this a file grabbed from nimbus-shared, or have you grabbed it then modified it? (ISTR we discussed modifying it then asking experimenter for a schema file).

Reply:
Ya in the latest patch I modified the schema to allow for all the features to be in one json (by making them an arbitrary number of key-value pairs, where the value abides by the schema that I got from moz-central)